source/extensions/clusters/redis/BUILD

```diff
@@ -36,12 +37,13 @@ envoy_cc_library(
     ],
 )
 
-envoy_cc_library(
+envoy_cc_extension(
     name = "redis_cluster",
     srcs = [
         "redis_cluster.cc",
         "redis_cluster.h",
     ],
+    security_posture = "robust_to_untrusted_downstream",
```

Comment:
I for now this assumes trusted everything, as we don't yet have fuzzers, etc. cc @HenryYYang